versions/3.0.3.md

```diff
@@ -1891,13 +1891,13 @@ $request.body#/successUrls/2 | http://clientdomain.com/medium
 $response.header.Location | http://example.org/subscription/1
 
 
-##### Callback Object Example
+##### Callback Object Examples
 
-The following example shows a callback to the URL specified by the `id` and `email` property in the request body.
+The following example uses the user provided `queryUrl` query string parameter to define the callback URL.
 
 ```yaml
-myWebhook:
```

Comment:
here we were conflating callbacks and webhooks, which could be confusing with Webhooks getting their own proposal: #1974.

Reply:
Hence, why I am not a fan of calling the new thing WebHooks.  Callbacks were designed to support Webhooks.

Reply:
Let's save discussions for naming of other things in the PRs for other things. This is callbacks and we should talk about callbacks and whether callbacks support webhooks nicely or not, we can be a bit less confusing here by sticking to the term callback consistently.